firebase-appdistribution/src/main/AndroidManifest.xml

```diff
@@ -35,6 +35,15 @@
         <activity android:name="com.google.firebase.appdistribution.internal.InstallActivity" android:exported="true" android:launchMode="singleTask"/>
 
         <activity android:name="com.google.firebase.appdistribution.internal.SignInResultActivity" android:exported="true">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW"/>
+                <category android:name="android.intent.category.DEFAULT"/>
+                <category android:name="android.intent.category.BROWSABLE"/>
+
+                <data android:scheme="appdistribution" android:path="/${applicationId}" android:host="authredirect" />
```

Comment:
Nit: could you rearrange this so that the order reflects the actual order in a URI: scheme, path, host